doc/newsboat.asciidoc

```diff
@@ -1262,14 +1262,110 @@ The macro prefix can be changed from the default "," to another key, e.g. "+"
 
 === Open Links with External Commands
 
-==== Switching Browser with Macros
+==== Using Browser
+
+The first step to open links outside of newsboat is to set a default
+<<browser,`browser`>> in your config file. It can be any command that you would
+normally run, such as a web browser or an executable script. For example:
+
+  browser firefox
+
+The browser command also takes an optional `%u` to specify at which point the
+URL sould be expanded. If `%u` is not set, the URL will be appended at the end of
+the command. Therefore, the following example is equivalent to the last one:
+
+  browser "firefox %u"
+
+To open any link with your browser, the versatile
+<<open-in-browser,`open-in-browser`>> operation can be used. Its behavior is
+specific to each dialog. Here is an overview of the different kinds of links it
+can open:
+
+- From the article list or the article view, open the URL associated with the current article
+- From the feed list, open the URL associated with the current feed
+- From the URL view, open any URL present in the content of the article
+- From the URL view, open an RSS enclosure URL (e.g. the audio file of a podcast)
+
+The default keybinding for `open-in-browser` is "o". An alternative operation,
+<<open-in-browser-and-mark-read,`open-in-browser-and-mark-read`>> also marks the
+article as read when in the article list, but doesn't work in the feed list.
+The default keybinding is "O".
+
+Articles usually contain links which point to different resources and websites.
+Newsboat detects those links and creates a list which can be seen at the bottom
+of the article view. To easily scroll through the list and choose a link to
+open, one can go to the URL view with the "U" key.
+
+Those links can also be opened directly from the article view.
+Relevant operations are <<one,`one`>> to <<zero,`zero`>>
+(digits written as words) and <<goto-url,`goto-url`>>.
+Keys "1" to "0" can be used to open URL 1 to 10. To open URLs above 10,
+start with the "#" key, then type the URL number, and press "Enter"
+(note that because this operation is interactive, it cannot be used in macros).
```

Comment:
Well, this is fantastic news! All this time I thought I was limited to URL 1-10 inside macros.
I bet I'm not the only one who will miss that. I replaced the note with a small explanation.

Reply:
Did you check the code, though? I'm really not sure if it works across all operations, and I'd rather not document something I'm unsure about.

Reply:
I've been very specific with the new explanation:

> To use goto-url inside a macro, simply append the URL (e.g. `goto-url 11`).

I don't think it could be misleading. Plus, I did test it in a macro and it works fine.

Reply:
I still feel uneasy about it, but okay, let's do it your way for once :)

Although not quite: please change it to say "append the URL's number" or something like that. "11" is not a URL ;)

Reply:
If something I wrote doesn't work as intended, I sure hope someone will step up and file an issue.
Fixed.